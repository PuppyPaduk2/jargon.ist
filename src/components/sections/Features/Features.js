// @flow

import React from 'react';

import { Box } from '../../ui';

import read from '../../../img/read.svg';
import learn from '../../../img/learn.svg';
import contribute from '../../../img/contribute.svg';

const Features = () => (
  <div className="row">
    <div className="col col--lg-4 u-gap-bottom@md-down">
      <Box className="u-text-center">
        <figure>
          <img src={read} alt="oku" className="u-img-fluid" />
        </figure>

        <h3 className="u-color-primary u-gap-top-medium u-clear-gap-bottom">oku</h3>
        <p>
          Programlama dilleri veya bilgisayar bilimlerinin temel prensipleri hakkındaki terimleri
          oku.
        </p>
      </Box>
    </div>

    <div className="col col--lg-4 u-gap-bottom@md-down">
      <Box className="u-text-center">
        <figure>
          <img src={learn} alt="öğren" className="u-img-fluid" />
        </figure>

        <h3 className="u-color-primary u-gap-top-medium u-clear-gap-bottom">öğren</h3>

        <p>
          Öğrendiklerin sayesinde Türkçe ya da İngilizce kaynakları okurken konuya daha çabuk hakim
          ol.
        </p>
      </Box>
    </div>

    <div className="col col--lg-4">
      <Box className="u-text-center">
        <figure>
          <img src={contribute} alt="katkıda bulun" className="u-img-fluid" />
        </figure>

        <h3 className="u-color-primary u-gap-top-medium u-clear-gap-bottom">katkıda bulun</h3>

        <p>
          Kendi başına veya topluluktan öğrendiğin jargonları jargon.ist’te paylaş, herkese yardımda
          bulun.
        </p>
      </Box>
    </div>
  </div>
);

export default Features;
