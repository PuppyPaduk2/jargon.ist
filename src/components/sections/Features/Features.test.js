// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Features from './Features';

it('renders correctly', () => {
  const wrapper = shallow(<Features />);

  expect(wrapper).toMatchSnapshot();
});
