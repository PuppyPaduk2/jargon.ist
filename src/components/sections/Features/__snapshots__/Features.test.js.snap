// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col col--lg-4 u-gap-bottom@md-down"
  >
    <Box
      className="u-text-center"
    >
      <figure>
        <img
          alt="oku"
          className="u-img-fluid"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </figure>
      <h3
        className="u-color-primary u-gap-top-medium u-clear-gap-bottom"
      >
        oku
      </h3>
      <p>
        Programlama dilleri veya bilgisayar bilimlerinin temel prensipleri hakkındaki terimleri oku.
      </p>
    </Box>
  </div>
  <div
    className="col col--lg-4 u-gap-bottom@md-down"
  >
    <Box
      className="u-text-center"
    >
      <figure>
        <img
          alt="öğren"
          className="u-img-fluid"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </figure>
      <h3
        className="u-color-primary u-gap-top-medium u-clear-gap-bottom"
      >
        öğren
      </h3>
      <p>
        Öğrendiklerin sayesinde Türkçe ya da İngilizce kaynakları okurken konuya daha çabuk hakim ol.
      </p>
    </Box>
  </div>
  <div
    className="col col--lg-4"
  >
    <Box
      className="u-text-center"
    >
      <figure>
        <img
          alt="katkıda bulun"
          className="u-img-fluid"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </figure>
      <h3
        className="u-color-primary u-gap-top-medium u-clear-gap-bottom"
      >
        katkıda bulun
      </h3>
      <p>
        Kendi başına veya topluluktan öğrendiğin jargonları jargon.ist’te paylaş, herkese yardımda bulun.
      </p>
    </Box>
  </div>
</div>
`;
